# Drone location lservice
# To fetch current location of one drone

int8 active_id
---
# position array [x, y, z, r, p, y] USI
float64[] position
